sdk/eventhubs/azure-messaging-eventhubs-stress/README.md

```diff
@@ -0,0 +1,300 @@
+# Azure Event Hubs Stress test client library for Java
+
+Represents stress tests for Event Hubs client.
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk_link], version 8 or later.
+- [Maven][maven]
+- [Docker][docker]
+- [Kubectl][kubectl]
+- [Helm][helm]
+- [Azure CLI][azure_cli]
+- [Powershell 7.0+][powershell]
+
+Make sure you have permission to access subscription `Azure SDK Developer Playground` and resource group `rg-stress-cluster-test`.
+
+```shell
+# Authenticate to Azure
+az login
+
+# Download the kubeconfig for the cluster (creates a 'context' named 'stress-test')
+az aks get-credentials --subscription "Azure SDK Developer Playground" -g rg-stress-cluster-test -n stress-test
```

Comment:
Preferably we route people towards the deploy script with `-Login` only, that way if we change the cluster location/name it won't break people's workflows. But I can also see how it's useful to document some of the underlying commands for testing access, etc.

Reply:
Oh, that's great. Then I can remove this part. I couldn't have the chance to see what the error is when an un-authenticated person run that command.

Reply:
It looks something like this for me:

```
$ ./eng/common/scripts/stress-testing/deploy-stress-tests.ps1 -Login
Using module powershell-yaml with version 0.4.2.
Logging in to subscription, cluster and container registry
Listing AKS cluster in Azure SDK Developer Playground/rg-stress-cluster-test

==> az aks list -g rg-stress-cluster-test --subscription Azure SDK Developer Playground -o json

ERROR: AADSTS70043: The refresh token has expired or is invalid due to sign-in frequency checks by conditional access. The token was issued on 2022-05-25T21:53:38.2830291Z and the maximum allowed lifetime for this request is 43200.
Trace ID: 55e2ee84-b016-4dd3-b612-2bc94cf21000
Correlation ID: bc61b330-123d-4471-8bf1-e35a513bb967
Timestamp: 2022-05-26 17:56:58Z
To re-authenticate, please run:
az login --scope https://management.core.windows.net//.default
Run: /home/ben/sdk/azure-sdk-tools/eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1:24
Line |
  24 |      run @args
     |      ~~~~~~~~~
     | Command 'az aks list -g rg-stress-cluster-test --subscription Azure SDK Developer Playground -o json' failed with code: 1
```